assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -196,35 +199,40 @@ const Cart = () => {
 									// see: https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/1606
 									'location'
 								) }
-								selected={ selectedShippingRate }
-								renderOption={ ( option ) => ( {
+									selected={ selectedShippingRate }
+									renderOption={ ( option ) => ( {
 									label: option.name,
 									value: option.rate_id,
 									description: (
 										<Fragment>
 											{ option.price && (
-												<FormattedMonetaryAmount
-													currency={ getCurrencyFromPriceResponse(
-														option
+											<FormattedMonetaryAmount
+												currency={ getCurrencyFromPriceResponse(
+																option
+															) }
+												value={
+																option.price
+															}
```

Comment:
The indentation here doesn't seem quite right. No biggie if it passes the linter (not a blocker), but might be a hint to refactor to reduce nesting/indentation. There's 8 tabs of indentation when `ShippingRatesControl` starts ... pretty deep :)

Assigning the shipping calculator content to a variable might be cleaner, e.g. 

```js
const shippingCalculationsOptions = (
  <fieldset className="wc-block-cart__shipping-options-fieldset">
  // ...
);
```